src/components/mdx/article-series-list.tsx

```diff
@@ -21,6 +21,7 @@ const ArticleSeriesList: React.FC<
         {articles.map((article: any, i: number) => {
           return (
             <Link
+              key={i}
```

Comment:
an error I found in the console

Reply:
you shouldn't use index for key in general and use article slug or similar instead
```suggestion
              key={article.slug}
```